data-prepper-core/src/main/java/com/amazon/dataprepper/pipeline/server/DataPrepperServer.java

```diff
@@ -45,70 +25,9 @@
 
     @Inject
     public DataPrepperServer(
-            final DataPrepperConfiguration dataPrepperConfiguration,
-            final PluginFactory pluginFactory,
-            final DataPrepper dataPrepper,
-            final CompositeMeterRegistry systemMeterRegistry
+            final HttpServer server
     ) {
-        final int port = dataPrepperConfiguration.getServerPort();
-        final boolean ssl = dataPrepperConfiguration.ssl();
-        final String keyStoreFilePath = dataPrepperConfiguration.getKeyStoreFilePath();
-        final String keyStorePassword = dataPrepperConfiguration.getKeyStorePassword();
-        final String privateKeyPassword = dataPrepperConfiguration.getPrivateKeyPassword();
-
-        final PluginModel authenticationConfiguration = dataPrepperConfiguration.getAuthentication();
-        final PluginSetting authenticationPluginSetting;
-
-        if (authenticationConfiguration == null || authenticationConfiguration.getPluginName().equals(DataPrepperCoreAuthenticationProvider.UNAUTHENTICATED_PLUGIN_NAME)) {
-            LOG.warn("Creating data prepper server without authentication. This is not secure.");
-            LOG.warn("In order to set up Http Basic authentication for the data prepper server, go here: https://github.com/opensearch-project/data-prepper/blob/main/docs/core_apis.md#authentication");
-        }
-
-        if(authenticationConfiguration != null) {
-            authenticationPluginSetting =
-                    new PluginSetting(authenticationConfiguration.getPluginName(), authenticationConfiguration.getPluginSettings());
-        } else {
-            authenticationPluginSetting =
-                    new PluginSetting(DataPrepperCoreAuthenticationProvider.UNAUTHENTICATED_PLUGIN_NAME, Collections.emptyMap());
-        }
-
-        final DataPrepperCoreAuthenticationProvider authenticationProvider = pluginFactory.loadPlugin(DataPrepperCoreAuthenticationProvider.class, authenticationPluginSetting);
-        final Authenticator authenticator = authenticationProvider.getAuthenticator();
-
-        try {
-            if (ssl) {
-                LOG.info("Creating Data Prepper server with TLS");
-                this.server = createHttpsServer(port, keyStoreFilePath, keyStorePassword, privateKeyPassword);
-            } else {
-                LOG.warn("Creating Data Prepper server without TLS. This is not secure.");
-                LOG.warn("In order to set up TLS for the Data Prepper server, go here: https://github.com/opensearch-project/data-prepper/blob/main/docs/configuration.md#server-configuration");
```

Comment:
Please add these warnings back. They are necessary to inform our users that their configuration is insecure.